File: java/GraphLClient/src/main/java/com/sdl/web/Response/IGraphQLResponse.java
package com.sdl.web.Response;


import java.util.List;

public interface IGraphQLResponse {
    Object Data = null;
    List<GraphQLError> Errors = null;

    void setData(Object data);
    Object getData();

    void setErrors(List<GraphQLError> errors);
    List<GraphQLError> getErrors();
}

File: net/src/Sdl.Web.PublicContentApi/GraphQLClient/Response/IGraphQLResponse.cs
using System.Collections.Generic;
using Sdl.Web.HttpClient;

namespace Sdl.Web.GraphQLClient.Response
{
    /// <summary>
    /// Represents the GraphQL respinse
    /// </summary>
    public interface IGraphQLResponse
    {
        /// <summary>
        /// GraphQL Data
        /// </summary>
        dynamic Data { get; set; }

        /// <summary>
        /// GraphQL Errors
        /// </summary>
        List<GraphQLError> Errors { get; set; }

        /// <summary>
        /// Response headers
        /// </summary>
        HttpHeaders Headers { get; set; }
    }
}
